src/theory/quantifiers/bv_inverter.cpp

```diff
@@ -530,9 +570,80 @@ static Node getScBvUrem(bool pol,
       }
     }
   }
-  else
+  else  /* litk == BITVECTOR_SGT  */
   {
-    return Node::null();
+    if (idx == 0)
+    {
+      Node z = bv::utils::mkZero(w);
```

Comment:
`z` is used a lot, you can move it to the top (outside of if/else blocks).

Reply:
It's not used in every branch, that's why.